speechbrain/nnet/activations.py

```diff
@@ -33,15 +35,19 @@ class Softmax(torch.nn.Module):
     torch.Size([10, 50, 40])
     """
 
-    def __init__(self, apply_log=False, dim=-1, reshape=True):
+    def __init__(
+        self, apply_log=False, dim=-1, reshape=True, dtype=torch.float32
+    ):
         super().__init__()
 
         if apply_log:
-            self.act = torch.nn.LogSoftmax(dim=dim)
+            self.act = F.log_softmax
```

Comment:
This should be changes as it does not allow users to specify the dimension on which softmax should be applied.

Reply:
(there is something similar below)

Reply:
> This should be changes as it does not allow users to specify the dimension on which softmax should be applied.

Are you sure? Below, in the fwd,  I used `x_act = self.act(x, dim=self.dim, dtype=self.dtype)` so I do specify the dim which can be changed in the `self.__init__` method.